modules/tinymce/src/plugins/table/main/ts/Plugin.ts

```diff
@@ -7,6 +7,7 @@
 
 import Editor from 'tinymce/core/api/Editor';
 import PluginManager from 'tinymce/core/api/PluginManager';
+import { Selections } from '@ephox/darwin';
```

Comment:
Import out of order... again I'm not why the linter isn't complaining about this. I'll have to look into that.